src/children/Any.hack

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\XHP\ChildValidation;
+
+final class Any implements Constraint {
+  public function legacySerialize(): mixed {
+    return 1;
```

Comment:
This is intentionally the same as LegacyConstraintType::ANY, right? Might as well reuse that constant, then.

Reply:
It's a bit of a coincidence: `__xhpAttributeDeclaration()` returns `0|1|expr`, and `1` happens to have the same meaning as it does a constraint type